cadcdata/cadcdata/storageinv.py

```diff
@@ -748,7 +809,7 @@ def cadcput_cli():
                                      os.path.basename(file))
         else:
             file_id = args.identifier
-        logger.info('PUT {} -> {}'.format(file, file_id))
+        logger.debug('PUT {} -> {}'.format(file, file_id))
```

Comment:
This is probably one of the locations where I wouldn't change the logging level, since it's a CLI, and not a library, but your call.